fluentd/vendored_gem_src/fluent-plugin-prometheus/lib/fluent/plugin/in_prometheus_tail_monitor.rb

```diff
@@ -77,9 +84,18 @@ def update_monitor_info
           # Access to internal variable of internal class...
           # Very fragile implementation
           pe = watcher.instance_variable_get(:@pe)
+          totalbytesread = watcher.instance_variable_get(:@totalbytesread)
+          totalbytesavailable = watcher.instance_variable_get(:@totalbytesavailable)
+          maxfsize = watcher.instance_variable_get(:@maxfsize)
+          countonrotate = watcher.instance_variable_get(:@countonrotate)
           label = labels(info, watcher.path)
           @metrics[:inode].set(label, pe.read_inode)
           @metrics[:position].set(label, pe.read_pos)
+          @metrics[:maxfsize].set(label, maxfsize)
+          @metrics[:countonrotate].set(label, countonrotate)
+          @metrics[:totalbytesread].set(label, totalbytesread)
+          @metrics[:totalbytesavailable].set(label, totalbytesavailable)
+          #@log.info "IN PROMETHEUS PLUGIN pr.read_inode and pe.read_pos #{pe.read_inode} #{pe.read_pos} maxfsize #{maxfsize} countonrotate #{countonrotate} totalbytesread #{totalbytesread} totalbytesavailable #{totalbytesavailable} "
```

Comment:
Consider change to log.debug and remove comment

Reply:
restored back all changes done in vendored_gem_src as that is not supposed to be changed.